vixos/template_nix.py

```diff
@@ -24,7 +28,8 @@ def generate_nix_user(package: str, executable: str) -> str:
   imports = [
     <nixpkgs/nixos/modules/virtualisation/qemu-vm.nix>
     ./managed.nix
-    ../local.nix
+    ./local.nix
+    ./global.nix
   ];
 
   environment.systemPackages = [ appRunner pkgs.%s ];
```

Comment:
this should be in the user conigurable section (or at least configurable by the user). User should also have a way to change the entrypoint, probably.

Reply:
Giving user ability to overwrite entrypoint is not a trivial task, as vixos depends on it. Just go back to my "patching files which user edited" rant.

Reply:
don't have the energy to argue now. I guess we can merge this for now. I'll keep editing default.nix (because I need to install more packages to my VMs, for example) because this is all temporary anyway and most likely will change in the future many times.